public/model.js

```diff
@@ -161,16 +161,27 @@ window.model = (() => {
             this.rendered_description = data.rendered_description;
             this.stream_post_policy = data.stream_post_policy;
             this.subscribers = data.subscribers;
-
-            // our fields
-            this.topics = [];
         }
+
         get id() {
             return this.stream_id;
         }
         set id(id) {
             this.stream_id = id;
         }
+
+        get subs() {
+            const subs = [];
+            this.subscribers.forEach((e) => {
```

Comment:
Does the var `e` have two different meanings here?

Reply:
No, its consistent. Moving this one to chat.